### DISCUSSION TITLE
Why I’m not eligible? I have only one wallet and top 1%

### DISCUSSION THREAD

**Discussion by Berkay6600** [zkSync-Community-Hub/zksync-developers#585] [General]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

I am deeply shocked, I have been farming for months, fulfill all the criteria, Gitcoim well over 20 points and have been doing that for 6 months, always had over 0.01 Eth in my wallet and above all I only have one wallet, put all my money in it and now nothing, do you have a clear conscience?
![IMG_0471](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/15f8beb6-3354-4740-9933-0ca412deb9d7)
![IMG_0472](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/7233479d-0a5c-4b7f-823f-2329230be4b8)
![IMG_0473](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/eef0dc7a-4069-4be6-add5-ae2b9018eca5)
![Uploading IMG_0477.png…]()

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

**Reply by Berkay6600** [reply]
Hello, I obviously meet the criteria. 75 times Paymaster, more than 20 smart contracts, traded 10 different ERC tokens?

Moreover my to my TWB:

10*1000 +50*120: 366= 43.7 dollars

In addition I have Gitcoin 43 Humainty Scores

Multiple NFT

Burned fees of $300.

I politely request a detailed answer


![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/63751c05-c4e8-4259-90c8-1cfbddf874fa)

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/714846cf-d1fc-4175-a6ef-f31f0a945d79)


![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/2251775f-1d0f-439a-a289-007d84cb74dd)

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/eb768db9-9bff-4b18-9818-bd5c96a7354a)

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/4eaa39c5-a2c4-4f7b-a0b1-ba185c73b5cd)


![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/147265761/392fa76f-f839-4169-b479-6782ac73421d)

**Reply by Berkay6600** [reply]
Furthermore I did more than 8 withdrawls to Zksync.
